site/_posts/2025-04-15-manageiq-radjabov-ga-announcement.md

```diff
@@ -0,0 +1,64 @@
+    - Create snapshot for a Virtual Machine [[#267]](https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/267)
+- Core
+  - Added new product feature that controls who can see Request Logs [[#22701]](https://github.com/ManageIQ/manageiq/pull/22701)
+  - Upgrade Ruby to 3.3 [[#23142]](https://github.com/ManageIQ/manageiq/pull/23142)
+  - Upgrade Rails to 7.0 [[#22052]](https://github.com/ManageIQ/manageiq/issues/22052)
+  - Workflows
+    - Add provisioning to workflows [[#23039]](https://github.com/ManageIQ/manageiq/pull/23039)
+    - Use Floe wait for event driven updates [[#109]](https://github.com/ManageIQ/manageiq-providers-workflows/issues/109)
+    - Expose requester to workflows [[#101]](https://github.com/ManageIQ/manageiq-providers-workflows/issues/101)
+    - Add an embedded_ansible builtin method [[#86]](https://github.com/ManageIQ/manageiq-providers-workflows/pull/86)
+    - Add provision_execute builtin method [[#81]](https://github.com/ManageIQ/manageiq-providers-workflows/pull/81)
+    - Rename 'builtin' scheme to 'manageiq' [[#96]](https://github.com/ManageIQ/manageiq-providers-workflows/pull/96)
+    - Provide out of the box workflows [[#744]](https://github.com/ManageIQ/manageiq-content/pull/744), [[#750]](https://github.com/ManageIQ/manageiq-content/pull/750)
+    - [ASL] Add Intrinsic Functions [[floe#64]](https://github.com/ManageIQ/floe/issues/64)
+    - [ASL] Implement Map state [[floe#184]](https://github.com/ManageIQ/floe/pull/184)
+    - [ASL] Implement Parallel State [[floe#287]](https://github.com/ManageIQ/floe/issues/287)
+- UI
+  - Many page conversions from Angular to React
+  - Many page conversions from HAML to React [[#9396]](https://github.com/ManageIQ/manageiq-ui-classic/issues/9396), [[#9399]](https://github.com/ManageIQ/manageiq-ui-classic/issues/9399), [[#9400]](https://github.com/ManageIQ/manageiq-ui-classic/issues/9400)
+- API
+  - Add `/api/providers/#/metric_rollups` [[#22827]](https://github.com/ManageIQ/manageiq/issues/22827)
+- Build
+  - [Appliances] Upgrade to EL9 [[#563]](https://github.com/ManageIQ/manageiq-appliance-build/pull/563)
+  - [Appliances] Set Ruby stream to 3.3 [[#578]](https://github.com/ManageIQ/manageiq-appliance-build/pull/578)
+  - [Containerized] Upgrade to UBI9 [[#848]](https://github.com/ManageIQ/manageiq-pods/issues/848)
+  - [Operator] Upgrade the operator to UBI9 [[#1055]](https://github.com/ManageIQ/manageiq-pods/pull/1055)
+- Developer
+  - Update Yarn to v4.3.1 [[#23072]](https://github.com/ManageIQ/manageiq/pull/23072)
+  - Default PostgreSQL encryption to scram-sha-256 [[#388]](https://github.com/ManageIQ/manageiq-appliance/pull/388)
+
+**IMPORTANT**: The following are potentially breaking changes and deprecations
+
+- \*\***BREAKING**\*\*
+  - CentOS Stream 8 is no longer supported.
+  - Removal of PostgreSQL 13 [[#23065]](https://github.com/ManageIQ/manageiq/pull/23065)
```

Comment:
Did you mean to say 10?  13 is the current version.

Reply:
Gah yes I meant 10. Can you push a commit for it?

Reply:
Done